### PR TITLE
New field: IMDb number of votes #29

### PR DESCRIPTION
**Pull request by kkayacan**
Added
- New field: IMDb number of votes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 3.7.2 (2018-02-11)
+### Added
+  - New field: IMDb number of votes
```
Comment (jreklund) [inline CHANGELOG.md]: Add a reference at the end, to the Issue.
`- New field: IMDb number of votes #29`

File: includes/movie.search.inc.php [hunk 2]
```diff
@@ -131,9 +131,9 @@
 	if($templateName === 'poster' || $templateName === 'postertitle')
 		$columns = array('`id`','`name`','`year`','`seen`','`own`','`favourite`','`tv`');
 	if($templateName === 'posterlist' || $templateName === 'listplot')
-		$columns = array('`id`','`name`','`year`','`duration`','`rating`','`languages`','`plotoutline`','`seen`','`own`','`favourite`','`tv`','`pg`');
+		$columns = array('`id`','`name`','`year`','`duration`','`rating`','`votes`','`languages`','`plotoutline`','`seen`','`own`','`favourite`','`tv`','`pg`');
```
Comment (jreklund) [inline includes/movie.search.inc.php]: This change aren't needed. Votes dosen't get used in collection (start page).

File: includes/movie.search.inc.php [hunk 3]
```diff
@@ -131,9 +131,9 @@
...
 	if($templateName === 'list')
-		$columns = array('`id`','`name`','`year`','`duration`','`rating`','`languages`','`seen`','`own`','`favourite`','`tv`','`pg`');
+		$columns = array('`id`','`name`','`year`','`duration`','`rating`','`votes`','`languages`','`seen`','`own`','`favourite`','`tv`','`pg`');
```
Comment (jreklund) [inline includes/movie.search.inc.php]: This change aren't needed. Votes dosen't get used in collection (start page).

**Comment by kkayacan** [comment]
I pushed your requests. I think that's all I have to do, right? I am novice on this Git and Github thing.

**Comment by jreklund** [comment]
That's all. I have now merged your pull request into the main projekt. Thanks!

**Comment by kkayacan** [comment]
Thank you very much. That was my first one. Means a lot 👍

**Pull request merged**